generate_cscc.R

```diff
@@ -526,3 +593,4 @@ for (.rcp in rcps){
 }
 print("end")
 
+write.table(cscc, file = file.path("data", paste0(dmg_ref, ".csv")), col.names = FALSE,row.names=FALSE, sep =",")
```

Comment:
Nothing should happen after the end statement. Why are you writing to the data folder? Also we have already saved this info in store_scc.